### DISCUSSION TITLE
How can I obtain a fully resolved configuration from a repository?

### DISCUSSION THREAD

**Discussion by onthe-hook** [gruntwork-io/knowledge-base#517] [Help] [answered]
Is it possible to obtain a terragrunt configuration from a repository with all the references resolved?  
The use case is to load a TG config to compare results against an older, non-TG config and/or an existing deployment.
The goal is to support replacing an older, non-TG config with a repository of TG configs.

In a nutshell, what I'm looking for is something equivalent to _rendered-json=f( git@github.com:repository/path/terragrunt.hcl )_

render-json seems closest, but it requires a fair bit of file management with  git clone, navigating the file tree and managing temporary files.   Is there not anything more direct?   

---

<ins datetime="2022-07-23T20:21:27Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109044">Tracked in ticket #109044</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
Unfortunately, this feature does not exist. We currently have no features that support running Terragrunt against remote configuration references. We recommend working around this by implementing a wrapper script that is dedicated to this task.

**Comment by onthe-hook** [comment]
OK, thank you.   

As source references in the .hcl files can be remote references, I was curious if arguments could be as well.
